packages/theme/src/divider.scss

```diff
@@ -0,0 +1,3 @@
+.puik-divider {
+  @apply text-primary-400;
```

Comment:
We can't modify the width or the thickness of the divider in props ?

Reply:
It's not specified in the doc => https://docs.uikit.prestashop.com/34f250d18/p/700681-divider/b/454f45
So I think it's good like that

Reply:
OK fine for me.